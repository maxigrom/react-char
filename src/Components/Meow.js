// @flow

import React from 'react';
import Avatar from '@material-ui/core/Avatar';

type Props = {
  className?: string,
}

const Meow = (props: Props) => (
  <Avatar alt="Meow!" className={props.className} src="./assets/favicon.jpg"/>
);

export default Meow;
