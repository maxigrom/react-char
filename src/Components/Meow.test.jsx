/* eslint-env jest */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Meow from './Meow';

describe('<Meow />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Meow />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders correctly', () => {
    const tree = renderer.create(<Meow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
