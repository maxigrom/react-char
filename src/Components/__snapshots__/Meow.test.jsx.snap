// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meow /> renders correctly 1`] = `
<div
  className="MuiAvatar-root-4"
>
  <img
    alt="Meow!"
    className="MuiAvatar-img-6"
    src="/assets/favicon.jpg"
  />
</div>
`;
